### PR TITLE
Make sure Ltac2 get cleaned too.

### PR DESCRIPTION
**Pull request by silene**

### CODE REVIEW

**Review by gares — APPROVED** [review]

**Comment by gares** [comment]
Thanks, I'll just wait for CI to complete before merging.

**Comment by ppedrot** [comment]
@gares ping

**Comment by gares** [comment]
@coqbot merge now please

**Pull request merged**
